build.gradle

```diff
@@ -14,9 +15,10 @@ apply plugin: 'com.mparticle.kit'
 android {
     defaultConfig {
         minSdkVersion 14
+        resValue "string", "apptentive_distribution_version", project.version
```

Comment:
For string constants, adding them to either the config or to a res file works, but could you move both of them to the same place, i.e either both of them in a res file, or both of them in the config?